src/api.ts

```diff
@@ -26,3 +26,9 @@ function get<T>(url: string): Promise<T> {
 export const getTodos = () => get<Todo[]>('/todos');
 
 export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+
+export const getActiveTodos = (): Promise<Todo[]> => {
+  return getTodos().then(todos =>
+    todos.filter(todo => todo.completed === false),
+  );
+};
```

Comment:
```suggestion
export const getActiveTodos = (): Promise<Todo[]> => {
  return getTodos().then(todos =>
    todos.filter(todo => !todo.completed),
  );
};
```
- I think you don't need this function
- Don't compare with false, `todo.completed` is a boolean